misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/CoverageReport.java

```diff
@@ -187,4 +224,57 @@ private void createReport(final IBundleCoverage bundleCoverage, ModuleCoverage m
         return filteredPathList;
     }
 
+    private void writeReport(final IBundleCoverage bundle) {
+
+        try {
+            final IReportVisitor visitor = createReportVisitor();
+
+            // Not sure what this does. Enable and test
+
+            final ExecFileLoader loader = new ExecFileLoader();
+            File execFile = coverageDir.resolve("ballerina.exec").toFile();
+            loader.load(execFile);
+
+            visitor.visitInfo(loader.getSessionInfoStore().getInfos(), loader.getExecutionDataStore().getContents());
+            visitor.visitBundle(bundle, getSourceLocator());
+            visitor.visitEnd();
+
+        } catch (IOException e) {
+        }
+
+    }
+
+    private IReportVisitor createReportVisitor() throws IOException {
+
+        IReportVisitor visitor = null;
+
+        final XMLFormatter formatter = new XMLFormatter();
+        visitor =
+                formatter.createVisitor(new FileOutputStream(target.getReportPath().resolve("report.xml").toString()));
+
+//        final HTMLFormatter formatter = new HTMLFormatter();
+//        visitor =
+//                formatter.createVisitor(
+//                new FileMultiReportOutput(target.getReportPath().resolve("report.html").toFile()));
```

Comment:
Since we dont need an html formatter for this, we can remove the commented portion of the code as well